pkg/config/config_test.go

```diff
@@ -40,6 +40,31 @@ func TestReadStringSlice(t *testing.T) {
 	Assert(t).AreEqual("c", s[2], "Third entry c")
 }
 
+func TestReadBool(t *testing.T) {
+	conf, err := readTestFile().ReadMap("bool")
+	Assert(t).IsNil(err, "bool should be a map")
+
+	v, err := conf.ReadBool("t")
+	if err != nil {
+		panic(err.Error())
+	}
+	Assert(t).AreEqual(v, true, "Mismatch")
+
+	v, err = conf.ReadBool("f")
+	if err != nil {
+		panic(err.Error())
+	}
+	Assert(t).AreEqual(v, false, "Mismatch")
+
+	v, err = conf.ReadBool("default")
+	// Expected error, defaulting to false
+	if err == nil {
+		panic(err.Error())
```

Comment:
careful, since only get here if `err` is `nil`, the `panic` will not contain an error as useful as desired

Reply:
Good point. I opened a new PR here: https://github.com/square/p2/pull/1099